pyproject.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
   "requests",
   "pydantic<2",
   "transformers==4.31.0",
+  "safetensors<0.3.2", # related issue: https://github.com/huggingface/safetensors/issues/314
```

Comment:
Is there any dependency in the core list pulling `safetensors` in? I believe you don't need to put it both here and in `inference`. It's only needed in the same group of the package that depends on it.

Reply:
It is a transformers' dependency.
So it probably makes sense to pin it only in the core dependencies...

Adapted the PR.